overlord/servicestate/quota_control_test.go

```diff
@@ -111,28 +159,50 @@ func checkQuotaState(c *C, st *state.State, exp map[string]quotaGroupState) {
 	}
 }
 
-func checkSvcAndSliceState(c *C, snapSvc string, slicePath string, sliceMem quantity.Size) {
+func shouldMentionSlice(resources quota.Resources) bool {
```

Comment:
I don't understand the name of this, as it's doing a full validate, the comment is not super clear to me

Reply:
Updated this and made the check explicit